### PR TITLE
Switch to new C helpers

### PR DESCRIPTION
**Pull request by MichaelS11**
After https://github.com/mattn/go-oci8/pull/286

### CODE REVIEW

**Comment by MichaelS11** [comment]
Can any of the old branches be deleted?

https://github.com/mattn/go-oci8/branches

**Comment by MichaelS11** [comment]
Good to go

File: README.md [hunk 1]
```diff
@@ -1,24 +1,37 @@
...
 
-This package can be installed with the go get command:
+Install Oracle full client or Instant Client:
 
-    go get github.com/mattn/go-oci8
+https://www.oracle.com/technetwork/database/database-technologies/instant-client/downloads/index.html
 
-You need to put `oci8.pc` like into your `$PKG_CONFIG_PATH`. `oci8.pc` should be like below.
+Install pkg-config, edit your package config file oci8.pc (examples below), then set environment variable PKG_CONFIG_PATH to oci8.pc file location
+(Or can use Go tag noPkgConfig then setup environment variables CGO_CFLAGS and CGO_LDFLAGS)
```
Comment (mattn) [inline README.md]:
```
`noPkgConfig`
```
quote with backtick?

File: connection.go [hunk 2]
```diff
@@ -207,31 +201,25 @@ func (conn *OCI8Conn) prepare(ctx context.Context, query string) (driver.Stmt, e
...
-		stmt,
+		(*C.OCIStmt)(*stmt),
 		conn.errHandle,
 		(*C.OraText)(unsafe.Pointer(pquery)),
 		C.ub4(C.strlen(pquery)),
 		C.ub4(C.OCI_NTV_SYNTAX),
 		C.ub4(C.OCI_DEFAULT),
 	); rv != C.OCI_SUCCESS {
-		C.OCIHandleFree(s, C.OCI_HTYPE_STMT)
+		C.OCIHandleFree(*stmt, C.OCI_HTYPE_STMT)
```
Comment (mattn) [inline connection.go]: Thank  your fixing my wrong here.

**Comment by mattn** [comment]
Thanks

**Pull request merged**